ydb/core/kqp/provider/yql_kikimr_gateway.cpp

```diff
@@ -120,6 +120,7 @@ void IKikimrGateway::BuildIndexMetadata(TTableMetadataResult& loadTableMetadataR
 
         TString error;
         auto indexType = TIndexDescription::ConvertIndexType(index.Type);
+        YQL_ENSURE(indexType != NKikimrSchemeOp::EIndexType::EIndexTypeGlobalVectorKmeansTree, "Vector index is not supported.");
```

Comment:
It's ok, but the problem not in "is not supported", it's more about this code works only for secondary index and we don't expect here some other index.
Other index probably should be handled differently in the other place.

Reply:
The whole function `BuildIndexMetadata` is not used at all.
So I don't want to refactor unused function in this PR.
I place `YQL_ENSURE` to prevent misusing it in the future.